zsftp/queries.py

```diff
@@ -0,0 +1,36 @@
+from zsftp.sftp import SFTP
+from zdynamodb import logger
```

Comment:
What is this import , why you are importing logger from zdynamo @manishprasad75

Reply:
@vibhav-zf  How did we miss this ?

Reply:
Changed